lib/usdUfe/ufe/UsdUndoPayloadCommand.h

```diff
@@ -39,10 +39,11 @@ class USDUFE_PUBLIC UsdUndoLoadUnloadBaseCommand : public Ufe::UndoableCommand
 
     void saveModifiedLoadRules() const;
 
-private:
+protected:
     const PXR_NS::UsdStageWeakPtr _stage;
     const PXR_NS::SdfPath         _primPath;
     PXR_NS::UsdLoadPolicy         _policy;
+    PXR_NS::UsdStageLoadRules     _undoRules;
```

Comment:
You probably want to update / add to https://git.autodesk.com/maya3d/maya-usd/blob/dev/test/lib/ufe/testPayloadCommands.py